test/features/ecs-service/ecs-services.feature

```diff
@@ -95,7 +95,7 @@ Feature: We are able to instantiate all aws_ecs_service resources
             #--------|-------------------------------|---------------------|-------|
             | create | aws_alb_listener_rule         | set_priority        |       |
             |        | aws_ecs_service               | awsvpc_all          |       |
-            |        | aws_ecs_task_definition       | default             |       |
+            |        | aws_ecs_task_definition       | fargate             |       |
```

Comment:
Why was this change necessary?

Reply:
It was necessary only because I couldn't figure out why the test was failing with the "default" name. I figured the change occurred with the 0.12 upgrade of the terraform code, but this may warrant further investigation.

Reply:
This is correct after reviewing the Terraform code.